miden-lib/build.rs

```diff
@@ -446,7 +444,7 @@ fn is_masm_file(path: &Path) -> io::Result<bool> {
 fn generate_kernel_error_constants(kernel_source_dir: &Path) -> Result<()> {
     // Because the error files will be written to ./src/errors, this should be a no-op if ./src is
     // read-only
```

Comment:
nit: I'd probably move this comment to doc comments for this function.

Reply:
Done.